docs/modules/user-guide/partials/proc_defining-environment-variables.adoc

```diff
@@ -24,34 +24,7 @@ components:
       mountSources: true
       command: ['sleep', 'infinity']
       env:
-        - name: GOPATH
+        - name: gopath
```

Comment:
Are the names of environment variables supposed to be lowercase? I ask because this contradicts the general convention of environment variables in the shell.

Reply:
I can't speak to the general convention of environment variables in the shell, but the api/validation doc for devfiles says names should be lowercasesd: https://github.com/devfile/api/blob/master/pkg/validation/validation-rule.md 

So I assume it's fine to have the name lowercased, but yes, a SME confirmation would be great.